packages/react-aria-components/docs/Menu.mdx

```diff
@@ -64,137 +64,135 @@ import {MenuTrigger, Button, Popover, Menu, MenuItem} from 'react-aria-component
+
+  max-height: inherit;
+  box-sizing: border-box;
+  overflow: auto;
+  padding: 2px;
+  margin: 0;
+  min-width: 150px;
+  box-sizing: border-box;
+  outline: none;
 
-    max-height: inherit;
-    box-sizing: border-box;
-    overflow: auto;
-    padding: 2px;
-    margin: 0;
-    min-width: 150px;
-    box-sizing: border-box;
+  .react-aria-Section:not(:first-child) {
+    margin-top: 12px;
+  }
+
+  .react-aria-Header {
+    font-size: 1.143rem;
+    font-weight: bold;
+    padding: 0 0.714rem;
+  }
+
+  .react-aria-Separator {
+    height: 1px;
+    background: var(--separator-color);
+    margin: 2px 4px;
+  }
+
+  .react-aria-MenuItem {
+    margin: 2px;
+    padding: 0.286rem 0.571rem;
+    border-radius: 6px;
     outline: none;
+    cursor: default;
+    color: var(--text-color);
+    font-size: 1.072rem;
+    position: relative;
+    display: grid;
+    grid-template-areas: "label kbd"
+                        "desc  kbd";
+    align-items: center;
+    column-gap: 20px;
+
+    &[data-focused] {
+      background: var(--highlight-background);
+      color: var(--highlight-foreground);
+    }
 
-    .react-aria-Section:not(:first-child) {
-      margin-top: 12px;
+    &[data-disabled] {
+      color: var(--text-color-disabled);
     }
 
-    .react-aria-Header {
-      font-size: 1.143rem;
+    [slot=label] {
       font-weight: bold;
-      padding: 0 0.714rem;
+      grid-area: label;
     }
 
-    .react-aria-Separator {
-      height: 1px;
-      background: var(--separator-color);
-      margin: 2px 4px;
+    [slot=description] {
+      font-size: small;
+      grid-area: desc;
     }
 
-    .react-aria-MenuItem {
-      margin: 2px;
-      padding: 0.286rem 0.571rem;
-      border-radius: 6px;
-      outline: none;
-      cursor: default;
-      color: var(--text-color);
-      font-size: 1.072rem;
-      position: relative;
-      display: grid;
-      grid-template-areas: "label kbd"
-                          "desc  kbd";
-      align-items: center;
-      column-gap: 20px;
-
-      &[data-focused] {
-        background: var(--highlight-background);
-        color: var(--highlight-foreground);
-      }
-
-      &[data-disabled] {
-        color: var(--text-color-disabled);
-      }
-
-      [slot=label] {
-        font-weight: bold;
-        grid-area: label;
-      }
+    & kbd {
+      grid-area: kbd;
+      font-family: monospace;
+      text-align: end;
+    }
 
-      [slot=description] {
-        font-size: small;
-        grid-area: desc;
+    &[data-selection-mode] {
+      padding-left: 24px;
+      &::before {
+        position: absolute;
+        left: 4px;
+        font-weight: 600;
       }
 
-      & kbd {
-        grid-area: kbd;
-        font-family: monospace;
-        text-align: end;
+      &[data-selection-mode=multiple][data-selected]::before {
+        content: '✓';
+        content: '✓' / '';
+        alt: ' ';
+        position: absolute;
+        left: 4px;
+        font-weight: 600;
```

Comment:
Do we need this css here anymore since its been moved down to the selection section?

Reply:
good catch, thanks